recipes/ciso8601/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "ciso8601" %}
+{% set version = "1.0.3" %}
+{% set sha256 = "c2a6b94a7e69338a6a17d04f6a68536fc9a4721bd98d68b61b5c519c3e061ec7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+  run:
+    - python
+
+test:
+  imports:
+    - ciso8601
```

Comment:
The tests are not packaged with source on pypi. Should I manually package the file, switch to using the github release, or just not worry about it?